src/config/internalLinkConfig.d.ts

```diff
@@ -0,0 +1,14 @@
+export interface InternalLinkConfig {
+    testmode?: boolean
+    autocompleteUrl?: string
+    keywordUrl?: string
+    shortDescriptionUrl?: string
+    axiosInstance?: any
+    previewUrl?: string
+}
+
+declare module '@ckeditor/ckeditor5-core/src/editor/editorconfig' {
+    interface EditorConfig {
+        internallink?: InternalLinkConfig
+    }
+}
```

Comment:
same as in math: what do you need this for? i dont see any usages
does it not work the old way?

Reply:
Same as `ckeditor5-math`: typescript would complain about missing types for `internallink` field in editor config.